tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml

```diff
@@ -164,45 +165,48 @@
+   example TSN Streams are simple applicaions over DetNet flows.  The specific
+   of this operation are discussed later in this document.
   </t>
 
 <figure anchor="fig_tsn_mpls_detnet" align="center"
 title="A TSN over DetNet MPLS Enabled Network">
 <artwork align="center"><![CDATA[
- TSN             Edge          Transit        Edge        TSN
- End System      Node           Node          Node        End System
-                (T-PE)         (LSR)          (T-PE)
-
++----------+                                             +----------+
+|TSN       | <-------- End to End TSN Service ---------> |TSN       |
+|Application                                             |Application
 +----------+  +.........+                   +.........+  +----------+
-|   Appl.  |<-:Svc Proxy:--End to End Svc.--:Svc Proxy:->|   Appl.  |
-+----------+  +---------+                   +---------+  +----------+
-|    TSN   |  |TSN| |Svc|<-- DetNet flow -->|Svc| |TSN|  |    TSN   |
+|          |  |   :<-Svc:                   :Svc->:   |  |          |
+|          |  |   | :Pxy:<-  DetNet Svc.  ->:Pxy: |   |  |          |
+|   TSN    |  |TSN| +---+                   +---+ |TSN|  |   TSN    |
+|          |  |   | |Svc|<-- DetNet flow -->|Svc| |   |  |          |
 +----------+  +---+ +---+    +----------+   +---+ +---+  +----------+
-|Forwarding|  |Fwd| |Fwd|    |Forwarding|   |Fwd| |Fwd|  |Forwarding|
-+------.---+  +--.+ +-.-+    +---.----.-+   +--.+ +-.-+  +----.-----+
-       :   Link  :    /  ,-----.  \   :  Link  :   /  ,-----.  \
-       +.........+    +-[  Sub  ]-+   +........+   +-[  TSN  ]-+
+|   L2     |  | L2| |Fwd|    |Forwarding|   |Fwd| |L2 |  |   L2     |
++------.---+  +-.-+ +-.-+    +---.----.-+   +--.+ +-.-+  +---.------+
+       :  Link  :     /  ,-----.  \   :  Link  :   /  ,-----. \
+       +........+     +-[  Sub  ]-+   +........+   +-[  TSN  ]-+
                         [Network]                    [Network]
                          `-----'                      `-----'
 
-        |<- TSN ->| |<------ DetNet MPLS ------>| |<-- TSN --->|
+                    |<------ DetNet MPLS ------>|
+        |<---------------------- TSN   --------------------->|
+
 ]]></artwork>
 </figure>
 
   <t>
-	In this example, edge nodes provide a service proxy function that 
-	"associates" the DetNet flows and native flows (i.e., TSN Streams)
-	at the edge of the DetNet domain. This ensures that the DN Flow is 
-	properly served at the Edge node (and inside the domain).
+        In this example, edge nodes provide a service proxy function that 
+        "associates" the DetNet flows and native flows (i.e., TSN Streams) at
+        the edge of the DetNet domain.  The TSN streams are treated as DetNet
+        Applicaions.  The service proxy behaves as a TSN relay node as far as
+        the TSN flows are concerned. For DetNet MPLS the service proxy presents
+        the TSN services as an application flow. 
   </t>

```

Comment:
Not fully happy with the changes. I would prefer something like this:
"TSN streams are treated as App-flows for DetNet. The whole DetNet domain behaves as a TSN relay node for the TSN streams. The service proxy behaves as a port of that TSN relay node. "
Also concerns on this sentence: "For DetNet MPLS the service proxy presents the TSN services as an application flow." It is more the TSN stream, not the TSN service. I would delete this sentence.

Reply:
OK as agreed per call